modules/codegen/src/main/scala/com/twilio/guardrail/generators/Http4sServerGenerator.scala

```diff
@@ -233,7 +236,7 @@ object Http4sServerGenerator {
                 None,
                 Some(
                   (
-                    q"urlForm.values.get(${arg.argName.toLit}).map(_.map(Json.fromString(_).as[$tpe]).sequence)",
+                    q"urlForm.values.get(${arg.argName.toLit}).flatMap(_.toList).traverse(Json.fromString(_).as[$tpe])",
```

Comment:
I'm very wary of `.values.get("foo").flatMap(_.toList)`, I haven't had time to test it to be certain, but my intuition is that `urlForm.values.get(...)` would be `Map`, which would mean `get` would return `Option[_]`, which would mean `flatMap(_.toList)` would fail.

If you're available to hit this codepath or otherwise test this construct in the REPL, I'd appreciate it. I'd like to get this out, I'd just like to get it out without breaking functionality (as I expect you can appreciate).

Please pardon the delay around this, I should have been more communicative.

Reply:
Deal, I'll try to do my best to give to a REPL example and a fix if needed :)

Reply:
@blast-hardcheese, instead of REPL I've added `multi` and `multiOpt` cases for `formData.yaml` and put more tests. So, I hope this will be a better proof than REPL ;)

Reply:
Thank you!